hub/constants.py

```diff
@@ -153,3 +153,6 @@
 DEFAULT_READONLY = (
     os.environ.get("HUB_DEFAULT_READONLY", "false").strip().lower() == "true"
 )
+
+# for chunks with number of samples in this range, partial requests will be used to retrieve the data
+PARTIAL_CHUNK_RANGE = range(2, 128)
```

Comment:
Could you please explain why exactly in this range?

Reply:
removed this, no longer used